centreon/www/class/centreonDB.class.php

```diff
@@ -514,7 +857,7 @@ public function isIndexExists(string $table, string $indexName): bool
         $statement->bindValue(':index_name', $indexName);
 
         $statement->execute();
-        return ! empty($statement->fetch(\PDO::FETCH_ASSOC));
+        return !empty($statement->fetch(\PDO::FETCH_ASSOC));
```

Comment:
in our coding style we add a space after the `!`

Reply:
ok thanks it's fixed for the file

Reply:
Isn't it covered by the coding-style?
If not, we should consider adding it to the list WDYT?

Reply:
I have added it to my todo list ;)